packages/@sfx/sayt-plugin/src/index.ts

```diff
@@ -1,20 +1,2 @@
-/**
- * This test comment _supports_ [Markdown and other fun stuff](https://typedoc.org/guides/doccomments/)
- */
-const testObj = {
-  label: 'test text',
-  number: 6,
-}
-
-export const testFunc = (data: DummyInterface) => {
-  return exports.getLabel(data).toUpperCase();
-}
-
-export const getLabel = (data: DummyInterface) => {
-  return data.label;
-}
-
-export interface DummyInterface {
-  label: string;
-  number: number;
-}
+export { default as SaytPlugin } from './sayt-plugin';
```

Comment:
I think that we should be testing that the file is actually exporting the SaytPlugin.

Reply:
This test case was added in. I'm resolving this now.